ada/testsuite/tests/c_api/unit_file_providers/test.out

```diff
@@ -1,5 +1,5 @@
-Calling ufp_get_file_from_name (some_field=42, kind=0) with name: "bar"
 Calling ufp_get_file_from_name (some_field=42, kind=0) with name: "foo"
+Calling ufp_get_file_from_name (some_field=42, kind=0) with name: "bar"
```

Comment:
I have no idea about this diff, is it okay?

Reply:
Probably. This test exposes internal implementation details that are not really something we guarantee, so it’s been updated several times already. As long as you understand why it changed…

Reply:
This test is kind of bothersome, but not enough for us to fix it I'd say ...